site/assets/scss/_sidebar.scss

```diff
@@ -44,19 +40,18 @@
     content: escape-svg($sidebar-collapse-icon);
   }
 
-  &.active {
-    .bd-sidenav-group-link::before {
-      transform: rotate(90deg);
-    }
+  [aria-expanded="true"] {
+    color: rgba($black, .85);
 
-    > .bd-sidenav-group-link {
-      color: rgba($black, .85);
+    &::before {
+      transform: rotate(90deg);
```

Comment:
If we keep the transition we might as well add a transition for the arrow. IIRC the default duration is `.35s`

But let's wait for @mdo to chime in first

Reply:
It feels weird to transition the height but not the rotation indeed.